examples/verifying_gfd/verifying_gfd1.py

```diff
@@ -0,0 +1,92 @@
+from pathlib import Path
+import matplotlib.pyplot as plt
+import matplotlib.image as mpimg
+import desbordante
```

Comment:
```suggestion

import desbordante
import matplotlib.pyplot as plt
import matplotlib.image as mpimg
```
PEP 8. `import`s should be in alphabetical order, with third-party libraries separated from STL libs by one line. Same for the other files.